HLTriggerOffline/Btag/src/HLTBTagPerformanceAnalyzer.cc

```diff
@@ -139,8 +141,17 @@ void HLTBTagPerformanceAnalyzer::analyze(const edm::Event& iEvent, const edm::Ev
 		}
 
 		for (auto & BtagJT: JetTag) {
+			std::map<std::string, bool> inmodule;
+			inmodule["HEP17"]=(BtagJT.first->phi() >= -0.87) && (BtagJT.first->phi() < -0.52) && (BtagJT.first->eta() > 1.3);
```

Comment:
wouldn't it be better to have a function for handling the HE slices ?
(instead of having these number hard coded)